3.evaluate_model/scripts/nbconverted/calculate_silhouette_metrics.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Calculate Silhouette Scores
+# 
+# - per phenotype
+# - per feature space
+
+# In[1]:
+
+
+import pathlib
+import numpy as np
+import pandas as pd
+
+from sklearn.decomposition import PCA
+from sklearn.metrics import silhouette_score, silhouette_samples
+
+import sys
+
+sys.path.append("../utils")
+from split_utils import get_features_data
+
+
+# In[2]:
+
+
+np.random.seed(1234)
```

Comment:
Recommend random seed `0` to be consistent with Way Lab standard in other projects.

Reply:
i will stick with 1234